app/views/camp/insert.php

```diff
@@ -0,0 +1,264 @@
+                                </div>
+                            <fieldset>
+                                <div class="panel panel-primary">
+                                    <div class="panel-heading" style="font-size: large; padding-bottom: 15px">
+                                        <b>Contact Details for Camp</b>
+                                        <div class="btn btn-info" id="show4" style="float: right">Hide</div>
+                                    </div>
+                                    <div class="panel-body" id="panelBody4">
+                                        <label for="phone_no">Phone number </label>
+                                        <input id="phone_no" class="form-control" placeholder="Enter Phone Number" name="phone_no" required /><br>
+                                        <label for="email">Email </label>
+                                        <input id="email" placeholder="Enter Email" class="form-control" type="email" name="email" required /><br>
+                                        <label for="mailing_list">Mailing list </label>
+                                        <input id="mailing_list" class="form-control" placeholder="Enter Mailing List" type="email" name="mailing_list" /><br>
+                                        <label for="mailing_address">Mailing address </label>
+                                        <textarea id="mailing_address" class="form-control" placeholder="Enter Mailing Address" name="mailing_address" ></textarea><br>
+                                        <label for="gis_url">GIS URL </label>
+                                        <input id="gis_url" class="form-control" placeholder="Enter GIS URL - iframe tag" tname="gis_url" /><br>
+                                    </div>
+                                </div>
+                            </fieldset>
+                            
+                            <div class="col-sm-8 col-md-4">
+                            </div>
+                            <div class="col-sm-8 col-md-4" style="padding-top: 12px;">
+                                <input class="btn btn-success btn-block" type="submit" name="submit" value="Add Camp" />
+                            </div>
+                        </form>
+                        <br>
+
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<!-- Toggle Script -->
+<script>
+    $('#organisation').autocomplete(
+        '<?php echo base_url(); ?>organisations/list_json',
+        'name',
+        $('#organisation_id'),
+        'id'
+    );
+
+    $("#menu-toggle").click(function(e) {
+        e.preventDefault();
+        $("#wrapper").toggleClass("toggled");
+    });
+
+    $("#show1").click(function(e) {
+        e.preventDefault();
+        $("#panelBody1").toggle("display");
+        $(this).text(($(this).text() == 'Hide') ? 'Show More' : 'Hide');
+    });
+
+    $("#show2").click(function(e) {
```

Comment:
@imjalpreet, I already merged this and missed these suggestion.. Are they fixed in maser? If not can you please fix it?